rpc/grpc/server/src/service.rs

```diff
@@ -13,6 +13,7 @@ const GRPC_SERVICE: &str = "grpc-service";
 
 pub struct GrpcService {
     net_address: NetAddress,
+    bps: u64,
```

Comment:
Optional: Is it possible that the underlying implementation will care about anything else in `config` other than bps? Maybe some other config later on that may be used to drive some other behavior. If so, it might be worth passing `config` here instead.

It's kinda strange to see `bps` being a specific parameter of a grpc service.

Reply:
This was saving a direct crate dependency but I also like the config approach. Let's do that.